polymer.json

```diff
@@ -14,6 +14,6 @@
     "src/ctree-user-screen/ctree-user-screen.html"
   ],
   "extraDependencies": [
-    "bower_components/webcomponentsjs/webcomponents-lite.min.js"
+    "bower_components/webcomponentsjs/*.js"
```

Comment:
I think we don't want to do this because it could make the download cost much larger.  Is there a specific reason for this?  It looks like maybe we should have just changed this to `webcomponents-lite.js` (not sure where the `.min` comes from)

Reply:
Yes, actually in new plymer-cli the build failed here